docker-compose/monitor/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.5"
+services:
+  jaeger:
+    networks:
+      - backend
+    image: jaegertracing/all-in-one:latest
```

Comment:
Is this file being tested via CI? If not, it might be better to use a fixed version.

Reply:
> Is this file being tested via CI?

Nope.

> it might be better to use a fixed version.

I'd like to consider one hypothetical scenario, keeping in mind that the ATM/Monitor feature is still under active development (the UI isn't ready yet).

We set the jaeger all-in-one tag to v1.24 here. If someone makes a change now and merges to `master` before v1.25 is released, if I understand correctly, will users need to wait until v1.26 to obtain/use the features just merged?